measure_extinction/utils/make_obsdata_from_model.py

```diff
@@ -482,14 +521,26 @@ def make_obsdata_from_model(
                 "g-",
             )
 
+            (indxs,) = np.where(rb_iue["NPTS"] > 0)
+            ax.plot(rb_iue["WAVELENGTH"][indxs].to(u.micron), rb_iue["FLUX"][indxs], "r-")
+
             (indxs,) = np.where(rb_nrc["NPTS"] > 0)
             ax.plot(rb_nrc["WAVELENGTH"][indxs].to(u.micron), rb_nrc["FLUX"][indxs], "c-")
 
             (indxs,) = np.where(rb_lrs["NPTS"] > 0)
             ax.plot(rb_lrs["WAVELENGTH"][indxs].to(u.micron), rb_lrs["FLUX"][indxs], "r:")
 
+            (indxs,) = np.where(rb_niriss["NPTS"] > 0)
+            ax.plot(rb_niriss["WAVELENGTH"][indxs].to(u.micron), rb_niriss["FLUX"][indxs], "r-")
+
+            (indxs,) = np.where(rb_nrc["NPTS"] > 0)
+            ax.plot(rb_nrc["WAVELENGTH"][indxs].to(u.micron), rb_nrc["FLUX"][indxs], "r-")
```

Comment:
This looks like a repeat of lines 527-528.

Reply:
Correct.  Removed.